package.json

```diff
@@ -296,6 +297,7 @@
     "bignumber.js": "^4.1.0",
     "bn.js": "^4.11.7",
     "bowser": "^2.11.0",
+    "browserify-zlib": "^0.2.0",
```

Comment:
Why is this package needed, what role does it play?

Reply:
Because of this: https://github.com/MetaMask/metamask-extension/blob/f129a239c5afc42d9474f6f2aa64e983fd27f024/.storybook/main.js#L60
In the past the storybook build was failing https://app.circleci.com/pipelines/github/MetaMask/metamask-extension/44737/workflows/0b92e650-e1d3-4a04-9018-8dbb29d775cf/jobs/1263432

Reply:
btw @brad-decker other PRs are also affected by this issue @zone-live commented: ie: 
https://github.com/MetaMask/metamask-extension/pull/18069
https://app.circleci.com/pipelines/github/MetaMask/metamask-extension/45502/workflows/cafd351b-c248-46e4-9924-ac0269e18a4c/jobs/1288105